### PR TITLE
MOB 1650 Increasing the speed of erasing text and fixes for both android/iOS

### PR DESCRIPTION
**Pull request by amanjeetsingh150**
## Proposed Changes
Increasing the speed of erasing text and by default erasing 50 characters

## Testing

https://user-images.githubusercontent.com/12881364/204239823-08d113bc-3766-4607-a2d2-e7a99ba45cec.mp4

https://user-images.githubusercontent.com/12881364/204239865-77805fd6-c9b5-48c3-84e7-ddd4545ec1a3.mp4

## Issues Fixed
https://github.com/mobile-dev-inc/maestro/issues/222

### CODE REVIEW

File: maestro-client/src/main/java/maestro/Driver.kt [hunk 1]
```diff
@@ -69,4 +69,6 @@ interface Driver {
     fun takeScreenshot(out: Sink)
 
     fun setLocation(latitude: Double, longitude: Double)
+
+    fun eraseAllText(charactersToErase: Int)
```
Comment (dmitry-zaitsev) [inline maestro-client/src/main/java/maestro/Driver.kt:73]: It isn't really "all text" if you give it a number of characters. Shall we just call it `eraseText`?

File: maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt [hunk 2]
```diff
@@ -354,12 +354,16 @@ data class PressKeyCommand(
...
+data class EraseTextCommand (
+    val charactersToErase: Int?,
 ) : Command {
 
     override fun description(): String {
-        return "Erase $charactersToErase characters"
+        return if (charactersToErase != null) {
+            "Erase $charactersToErase characters"
+        } else {
+            "Erase 50 characters by default"
```
Comment (dmitry-zaitsev) [inline maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt:365]: Let's keep the message as it was

Comment (dmitry-zaitsev) [inline maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt:365]: Let's say `Erase text` if `charactersToErase` is null

Comment (amanjeetsingh150) [reply maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt]: Done ✅

**Review by dmitry-zaitsev — APPROVED** [review]

**Pull request merged**